mudlet-mapper.xml

```diff
@@ -5620,7 +5608,131 @@ function mmp.clearspecials(ignorerooms)
       end
     end
   end
-end</script>
+end
+
+
+
+function mmp.gotoAreaID(areaid, number, dashtype)
+
+	if not areaid or not tonumber(areaid) then
+		mmp.echo(&quot;To where do you want to go?&quot;)
+		return
+	end
+	
+	areaid = tonumber(areaid)
+	
+	if not mmp.areatabler[areaid] then
+		mmp.echo(&quot;Invalid area ID selected&quot;)
+		return
+	end
+
+  local possibleRooms, shortestPath, shortestBorder = {}, {}, 0
```

Comment:
This and everything below seems like a direct copy of `mmp.gotoArea`. Could you make this a local function and use it for both functions? That way we don't need to duplicate code which causes trouble when a bug is found.